### PR TITLE
Add csslint and check-css target (requires npm install -d)

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
 var JSLINT = './node_modules/jshint/bin/hint',
+    CSSLINT = 'node node_modules/csslint/cli.js',
```
Comment (jbuck) [inline make.js]: `cli.js` already have a node hashbang, so this can be changed to `./node_modules/csslint/cli.js`

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/43135fd803b3575/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/43135fd803b3575/output.txt

Total script time: 0.27 mins

File: package.json [hunk 2]
```diff
@@ -69,7 +69,8 @@
     "express-browserid": "0.0.x",
     "config": "0.4.x",
     "mongoose": "2.5.x",
-    "stylus": "0.25.x"
+    "stylus": "0.25.x",
+    "csslint": "0.9.8"
```
Comment (jbuck) [inline package.json]: Change the version number to `0.9.x` so that we get bugfix updates

**Comment by humphd** [comment]
Fixed the bulk of the real errors and warnings (hundreds of them).

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://butyr.org:8000/2a86e31b90d7dd7/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/2a86e31b90d7dd7/output.txt

Total script time: 0.31 mins

**Comment by humphd** [comment]
OK, I fixed the issues you asked about, and we're down to 1 error (still lots of warnings, but that's OK).

**Comment by jbuck** [comment]
This has been superceded by other pull requests

**Pull request closed without merging**